docs/book/v3/file.md

```diff
@@ -89,46 +89,43 @@ for more information about encoding and its exceptions.
 
 The following set of options are supported:
 
-- `target` (string; default: `*`): Target filename or directory; the new name
-  of the source file.
-- `source` (string; default: `*`): Source filename or directory which will be
-  renamed. Used to match the filtered file with an options set.
+- `match` (string; default: `*`): File name pattern to check if this filter should be applied to the input.
+Read the [`fnmatch` documentation](https://www.php.net/manual/en/function.fnmatch.php) for more detail
```

Comment:
```suggestion
Read the [`fnmatch` documentation](https://www.php.net/manual/function.fnmatch.php) for more detail
```

Drops the `/en/` path segment - the PHP site will then auto-detect browser language